src/parser.c

```diff
@@ -487,6 +487,10 @@ static mqtt_parser_rc_t mqtt_parser_process(mqtt_parser_t* parser, mqtt_message_
         message->publish.content.length = parser->needs;
         message->publish.content.data   = NULL;
 
+        if(!parser->needs) {
+          goto DONE;
+	}
```

Comment:
Improve the indentation.

Reply:
You can run `make cs` to do the fixing for you. But this will require clang-format v5 or v6 to be installed on your system.

Reply:
Done.